process/cg-transition.html

```diff
@@ -1,102 +1,237 @@
+      transition of work to a Working Group.
+    </p>
+    <h3>
+      Is the work ready for standardization?
+    </h3>
+    <p>
+      Please see elsewhere in the Guide: <a href=
+      "https://www.w3.org/Guide/standards-track/">W3C Recommendation Track
+      Readiness Best Practices</a>.
+    </p>
+    <h3>
+      What is the target Working Group and the relation to the CG?
+    </h3>
+    <p>
+      If the Working Group does not yet exist, the Community Group can prepare
+      by <a href="https://www.w3.org/Guide/process/charter.html">drafting a
+      Working Group charter</a>. Otherwise, if the group exists but the new
+      work is not within its scope W3C needs to <a href=
+      "https://www.w3.org/Guide/process/charter.html">recharter the group</a>
+      to accommodate the new work.
+    </p>
+    <p>
+      The charter of the target Working Group should address two topics:
+    </p>
+    <ul>
+      <li>The list of any deliverables transferred from a Community Group.
+      </li>
+      <li>The nature of the working relationship between the Working Group and
+      the Community Group (if any) for any deliverable originated by the
+      Community Group.
+      </li>
+    </ul>
+    <p>
+      @@@TODO extract more from <a href=
+      "https://github.com/w3c/charter-drafts/issues/262">charter template
+      discussion</a>
+    </p>
+    <p>
+      Community Group chairs should keep the Community Group participants aware
+      of the progress on the transition roadmap, in particular if it involves
+      chartering or re-chartering a Working Group.
+    </p>
+    <h3>
+      Is there a record of Community Group consensus for the transition?
+    </h3>
+    <p>
+      It is useful to record whether and how the Community Group reached
+      consensus on the transition (in a manner consistent with its practices).
+      The consensus should include expectations about the working relationship
+      with the Working Group moving forward.
+    </p>
+    <h3>
+      Are IPR commitments in place?
+    </h3>
+    <p>
+      Community Groups have a "two-step IPR policy" so that people make
+      lightweight commitments when work starts and more comprehensive
+      commitments once work is mature.
+    </p>
+    <ul>
+      <li>Initially, a participant makes certain commitments regarding their
+      own Contribution under the <a href=
+      "https://www.w3.org/community/about/process/cla/">W3C Community
+      Contributor Licensing Agreeement (CLA)</a>.
+      </li>
+      <li>Later, a participant can make a voluntary commitment regarding the
+      entire specification under the <a href=
+      "https://www.w3.org/community/about/agreements/final/">W3C Community
+      Final Specification Agreement (FSA)</a>.
+      </li>
+    </ul>
+    <p>
+      Prior to the transition of work to a Working Group, a Community Group
+      should allow some time for participants to make their "Final Spec"
```

Comment:
I don't know whether we should make that a "should" - more specifically, my impression is that in most cases, the IPR commitments are secured without the FSA process by virtue of having contributors join the WG.

Reply:
Now discusses ipr commitments more clearly.